swc/swc.bzl

```diff
@@ -42,6 +42,7 @@ def swc(name, srcs = None, args = [], data = [], output_dir = False, swcrc = Non
           See https://swc.rs/docs/usage/cli#--source-maps--s
           True/False are automaticaly converted to "true"/"false" string values the cli expects.
         swcrc: label of a configuration file for swc, see https://swc.rs/docs/configuration/swcrc
+        out_dir: base directory for output files
```

Comment:
add "relative to the output directory for this package" ?

Reply:
looking at https://docs.aspect.build/bazelbuild/rules_nodejs/5.3.0/packages/typescript/docs.html#ts_project-out_dir for inspiration.